src/signature.ts

```diff
@@ -1,41 +1,79 @@
-const { ecdsaSign, ecdsaRecover, publicKeyConvert } = require('ethereum-cryptography/secp256k1')
+import { ecdsaSign, ecdsaRecover, publicKeyConvert } from 'ethereum-cryptography/secp256k1'
```

Comment:
There was no specific reason that we had done this `require` here, right? 🤔

Reply:
seems fine to me